.github/workflows/main.yml

```
@@ -57,16 +90,50 @@ jobs:
+    needs: [mediafusion_docker_build, kodi_build]
+    if: "!github.event.release.prerelease"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get previous tag
+        id: previoustag
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:65: Double quote to prevent globbing and word splitting

(shellcheck)